### ISSUE TITLE
Protein q-value in command line

### ISSUE THREAD

**Issue by chloeshep** [vdemichev/DiaNN#925]
Hi there, firstly thank you for making such a wonderful program! It is a pleasure to use.
I have a quick question about the command line function. Can I set the minimum number of peptides required for protein quantitation to 2? Some of my proteins are quantitated based on 1 peptide and I would like to be more stringent. 
For example, if a protein has 5 peptides in sample A but only 1 peptide in sample B, I would like it to be reported in sample A but not in sample B. I will then impute that missing value downstream of DIA-NN.
I can do this manually in Excel using the pg and pr matrix however I would prefer to do this in DIA-NN to avoid making mistakes!  
My apologies if this has been answered somewhere else. 
Thanks in advance!

**Comment by michaelsteidel** [comment]
Sure its good practice to replace experimental evidence by an imputation model?

I think the only way to to this based on report.tsv (but would need basic R skills).

**Comment by vdemichev** [comment]
No, not really possible at the moment. But this is very easy to do in R: just filter the dataset (main report) for protein IDs with >= 2 peptides detected and then use the iq R package to infer protein quantities.

But for stringent reporting a better idea might be to filter based on PG.Q.Value. For matrices this will be --matrix-spec-q command.

Yes, in Excel this is quite difficult. In general, I really recommend to try R, learning the basics for this kind of work can take maybe a day or two, and will speed up any kind of data analysis tremendously - it's like effortless compared to Excel.
